packages/create-amplify/src/npm_package_manager_controller.ts

```diff
@@ -28,9 +30,21 @@ export class NpmPackageManagerController implements PackageManagerController {
     if (type === 'dev') {
       args.push('--save-dev');
     }
-    await this.execa(this.executableName, args, {
-      stdio: 'inherit',
+    let aggregatedStdout = '';
+    const aggregatorStream = new stream.Writable();
+    aggregatorStream._write = function (chunk, encoding, done) {
+      aggregatedStdout += chunk;
+      done();
+    };
+
+    const childProcess = this.execa(this.executableName, args, {
+      stdin: 'inherit',
       cwd: this.projectRoot,
     });
+
+    childProcess?.stdout?.pipe(aggregatorStream);
```

Comment:
are we throwing `stderr` away?

Reply:
same as above, I'll add stderr.

Reply:
Done in [52a5eda](https://github.com/aws-amplify/amplify-backend/pull/595/commits/52a5eda31ca1d6dc5bfdca59e500c1ed746c0669)